features/reordering/reordering_placed_order.feature

```diff
@@ -0,0 +1,26 @@
+@reordering @ui
+Feature: Reordering previously placed order
+    In order to reorder the same order as a Customer placed before in the name of this Customer
```

Comment:
Maybe something less complicated? 😄`In order to reorder an order in the name of a Customer`